content/en/data_jobs/databricks.md

```diff
@@ -141,6 +170,14 @@ Optionally, you can also set other init script parameters and Datadog environmen
 
 {{< /tabs >}}
 
+### Restart existing clusters
+
+The init script installs the Agent when clusters start.
+
+Existing clusters, like all-purpose clusters or long-lived job clusters, must be restarted for the init script to install the Datadog Agent.
+
+All new clusters that start will use the init script (if the script is global or configured for the job cluster).
```

Comment:
```suggestion
```

Reply:
>```
>For scheduled jobs that run on job clusters, the init script gets installed automatically on the next run.
>```


@warrierr Why make this comment about new clusters starting specific to jobs? Any new node that comes online (IIUC) will use the now installed init script to install the agent, even nodes added due to auto-sizing up on existing clusters (I would guess anyway).

Reply:
I added this because I think it clears up to customers what they need to take action on. In the global init script case, it doesn't make sense to go update the job cluster config for batch Databricks jobs since it will get the global init on the next run. So customers only need to take action to restart streaming job clusters and all-purpose clusters.

> Any new node that comes online (IIUC) will use the now installed init script to install the agent, even nodes added due to auto-sizing up on existing clusters
True, but to confidently get all nodes to have the agent, and know it is in a good state, you need to restart the cluster. Not sure adding the technical nuance here around how it can partially get added to new nodes because of autoscaling is necessary.

Reply:
Change applied with 

>For scheduled jobs that run on job clusters, the init script **installs the Datadog Agent** automatically on the next run.